core/templates/dev/head/pages/exploration_editor/editor_tab/training_data_editor_directive.html

```diff
@@ -44,7 +44,7 @@ <h3>Training Data</h3>
   <div>
     <div class="add-new-answer">
       <p class="training-data-answer-label"> Add New Answer </p>
-      <test-interaction-panel state-content="stateContent" input-template="inputTemplate" on-submit-answer="submitAnswer(answer)">
+      <test-interaction-panel state-content="stateContent" input-template="inputTemplate">
```

Comment:
Make sure to test this manually. You can find it when editing TextInput (say) in the exploration editor of a published exploration (there should be a button related to training data).